pkg/config/storage.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import validator "gopkg.in/go-playground/validator.v9"
+
+// StorageConfig provides configs for various storage backends
+type StorageConfig struct {
+	CDN   *CDNConfig   `validate:""`
+	Disk  *DiskConfig  `validate:""`
+	GCP   *GCPConfig   `validate:""`
+	Minio *MinioConfig `validate:""`
+	Mongo *MongoConfig `validate:""`
+}
+
+func setStorageTimeouts(s *StorageConfig, defaultTimeout int) {
```

Comment:
can we not get rid of this function and just use the `default` struct tag instead?